cmd/tellstone/audit_decrypt.go

```
@@ -0,0 +1,193 @@
+		runAuditDecrypt(args[1:])
+	case "-h", "--help", "help":
+		printAuditUsage()
+	default:
+		fmt.Fprintf(os.Stderr, "unknown audit subcommand %q\n\n", args[0])
+		printAuditUsage()
+		os.Exit(1)
+	}
+}
+
+func printAuditUsage() {
+	fmt.Fprintln(os.Stdout, "tellstone audit — audit log utilities")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)